tools/pepsi/src/cargo/test.rs

```diff
@@ -0,0 +1,224 @@
+    )]
+    pub packages: Vec<String>,
+
+    /// Test all packages in the workspace
+    #[arg(long, help_heading = heading::PACKAGE_SELECTION)]
+    pub workspace: bool,
+
+    /// Exclude packages from the build
+    #[arg(
+        long,
+        value_name = "SPEC",
+        action = ArgAction::Append,
+        help_heading = heading::PACKAGE_SELECTION,
+    )]
+    pub exclude: Vec<String>,
+
+    /// Alias for workspace (deprecated)
+    #[arg(long, help_heading = heading::PACKAGE_SELECTION)]
+    pub all: bool,
+
+    /// Test only this package's library
+    #[arg(long, help_heading = heading::TARGET_SELECTION)]
+    pub lib: bool,
+
+    /// Test only the specified binary
+    #[arg(
+        long,
+        value_name = "NAME",
+        action = ArgAction::Append,
+        num_args=0..=1,
+        help_heading = heading::TARGET_SELECTION,
+    )]
+    pub bin: Vec<String>,
+
+    /// Test all binaries
+    #[arg(long, help_heading = heading::TARGET_SELECTION)]
+    pub bins: bool,
+
+    /// Test only the specified example
+    #[arg(
+        long,
+        value_name = "NAME",
+        action = ArgAction::Append,
+        num_args=0..=1,
+        help_heading = heading::TARGET_SELECTION,
+    )]
+    pub example: Vec<String>,
+
+    /// Test all examples
+    #[arg(long, help_heading = heading::TARGET_SELECTION)]
+    pub examples: bool,
+
+    /// Test only the specified test target
+    #[arg(
+        long,
+        value_name = "NAME",
+        action = ArgAction::Append,
+        help_heading = heading::TARGET_SELECTION,
+    )]
+    pub test: Vec<String>,
+
+    /// Test all tests
+    #[arg(long, help_heading = heading::TARGET_SELECTION)]
+    pub tests: bool,
+
+    /// Test only the specified bench target
+    #[arg(
+        long,
+        value_name = "NAME",
+        action = ArgAction::Append,
+        help_heading = heading::TARGET_SELECTION,
+    )]
+    pub bench: Vec<String>,
+
+    /// Test all benches
+    #[arg(long, help_heading = heading::TARGET_SELECTION)]
+    pub benches: bool,
+
+    /// Test all targets
+    #[arg(long, help_heading = heading::TARGET_SELECTION)]
+    pub all_targets: bool,
+
+    /// Test only this library's documentation
+    #[arg(long)]
+    pub doc: bool,
+
+    /// Compile, but don't run tests
+    #[arg(long)]
+    pub no_run: bool,
+
+    /// Run all tests regardless of failure
+    #[arg(long)]
+    pub no_fail_fast: bool,
+
+    /// Outputs a future incompatibility report at the end of the build (unstable)
+    #[arg(long)]
+    pub future_incompat_report: bool,
+
+    /// If specified, only run tests containing this string in their names
+    #[arg(value_name = "TESTNAME")]
+    pub test_name: Option<String>,
+
+    /// Arguments for the test binary
+    #[arg(value_name = "args", trailing_var_arg = true, num_args = 0..)]
+    pub args: Vec<String>,
+}
+
+impl CargoCommand for Arguments {
+    const SUB_COMMAND: &'static str = "build";
```

Comment:
This should be `test`, no?
```suggestion
    const SUB_COMMAND: &'static str = "test";
```

Reply:
oops, yes, done